lib/node_modules/@stdlib/array/complex64/benchmark/benchmark.fill.js

```diff
@@ -0,0 +1,51 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var Complex64 = require( '@stdlib/complex/float32' );
+var isComplex64Array = require( '@stdlib/assert/is-complex64array' );
+var pkg = require( './../package.json' ).name;
+var Complex64Array = require( './../lib' );
+
+
+// MAIN //
+
+bench( pkg+':fill', function benchmark( b ) {
+	var arr;
+	var i;
+
+	arr = new Complex64Array( 5 );
+
+	b.tic();
+	for ( i = 0; i < b.iterations; i++ ) {
+		arr = arr.fill( new Complex64( 1.0, 1.0 ) );
+		if ( typeof arr !== 'object' ) {
+			b.fail( 'should return an object' );
+		}
+	}
+	b.toc();
+	if ( !isComplex64Array( arr ) ) {
```

Comment:
```suggestion
		out = arr.fill( new Complex64( 1.0, 1.0 ) );
		if ( typeof out !== 'object' ) {
			b.fail( 'should return an object' );
		}
	}
	b.toc();
	if ( !isComplex64Array( out ) ) {
```

While `arr` and `out` are equal, assigning to the same variable within a benchmark loop is not standard convention. Instead, we have `arr` be consistently assigned, and allow another temporary variable take the assignment.

A decent optimizing compiler may omit the return value altogether, but this is not a given, and assigning to `out` may help thwart some optimizations which could interfere with what we are attempting to measure.